smdebug/tensorflow/keras.py

```diff
@@ -366,6 +385,90 @@ def _add_metric(self, metric_name, metric_value: tf.Tensor = None):
             coll.set_tensor_ref(TensorRef.from_non_graph_var(metric_name))
         self.tensor_to_collections[metric_name] = {coll}
 
+    def save_custom_tensor(self, tensor_name, tensor_value, collections_to_write):
+        for collection in collections_to_write:
+            self.custom_tensors_to_save[tensor_name] = (tensor_value, collection)
+
+    def _save_custom_tensors_post_step(self):
```

Comment:
description of what this function is for

Reply:
Done.